Makefile

```diff
@@ -5,8 +5,13 @@ obj-m += $(MODULE_NAME).o
 KERNEL_HEAD := $(shell uname -r)
 KERNELDIR := /lib/modules/$(KERNEL_HEAD)/build
 PWD := $(shell pwd)
+ARCH := $(shell uname -p)
+ifeq ($(ARCH),aarch64)
+KCPPFLAGS="-mcmodel=large"
```

Comment:
If we do not define `CONFIG_ARM64_ERRATUM_843419` on aarch64, can we install module success?

Reply:
After testing disable CONFIG_ARM64_ERRATUM_843419 on my ARM64, there is no error when install module.

Reply:
So we should check whether we enable `CONFIG_ARM64_ERRATUM_843419 ` instead of aarch64?

Reply:
yes, that's will be better, but it does't work if using
 ifeq ($(CONFIG_ARM64_ERRATUM_843419),y)
     do something
endif
Any other way to check this?

Reply:
```Makefile
MODULE_NAME		:= trace_noschedule

ifneq ($(KERNELRELEASE),)
obj-m			:= $(MODULE_NAME).o
ifeq ($(CONFIG_ARM64_ERRATUM_843419),y)
KCPPFLAGS		= "-mcmodel=large"
endif
else
PWD			:= $(shell pwd)
KERNEL_HEAD		:= $(shell uname -r)
KERNEL_DIR		:= /lib/modules/$(KERNEL_HEAD)/build

all:
	$(MAKE) KCPPFLAGS=$(KCPPFLAGS) -C $(KERNEL_DIR) M=$(PWD) modules

clean:
	$(MAKE) -C $(KERNEL_DIR) M=$(PWD) clean

install:
	sudo insmod $(MODULE_NAME).ko

remove:
	sudo rmmod $(MODULE_NAME)

endif
```

Does this work?

Reply:
It doesn't work, because $(CONFIG_ARM64_ERRATUM_843419) will always be null.

Reply:
Actually I think we use aarch64 is enough and no side effect if there is no better way to solve this.